src/CMakeLists.txt

```diff
@@ -462,6 +466,10 @@ ADD_SUBDIRECTORY ( Commands )
 ADD_SUBDIRECTORY ( Desktop )
 ADD_SUBDIRECTORY ( CommandLine )
 ADD_SUBDIRECTORY ( Tests )
+if (CMAKE_COMPILER_IS_GNUCC)
```

Comment:
Our debian package also currently builds on a bunch of other architectures (arm, mips, alpha...), so it would be good to add a check for x86 to the CMakeLists.txt around the simd and cpuinfo stuff.

The most robust way to do this might be try_compile on something using inline x86 assembly (the cpuinfo directory defines a project, so it should work for this purpose).